ycmd/http_client.py

```diff
@@ -169,3 +204,21 @@ def CppSemanticCompletionResults(server, path, row, col, contents, filetype='cpp
                                             line_num=row,
                                             column_num=col,
                                             contents=contents)
+
+
+def DefaultSettings():
```

Comment:
Maybe get default settings from ycmd dir? And allow to set path to this config in plugin's settings.